next-start-cluster.js

```diff
@@ -0,0 +1,47 @@
+const cluster = require('node:cluster');
```

Comment:
Need to remove cluster mode before merging

Reply:
done